components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/common/IdentityConnectorConfig.java

```diff
@@ -43,4 +43,6 @@
 
     Map<String, String> getDefaultPropertyValues (String[] propertyNames, String tenantDomain) throws IdentityGovernanceException;
 
+    Map<String, String> getPropertyTypeMapping();
```

Comment:
Better to add javadoc comments to methods.
Refer java.util.List interface. They have added method description for each method.
Then the overriding methods does not have to explicitly provide doc comments.